### PR TITLE
Add to CloudWatch tips and gotchas.

### PR DESCRIPTION
**Pull request by bgdnlp**

### CODE REVIEW

**Comment by jlevy** [comment]
Awesome. Great items, @bgdnlp ! LGTM.

@JCBarry want to do a 2nd review?

File: README.md [hunk 1]
```diff
@@ -808,15 +808,18 @@ CloudWatch
...
 * You can [publish your own metrics](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html) using the AWS API. [Incurs additional cost](https://aws.amazon.com/cloudwatch/pricing/).
 * You can stream directly from CloudWatch Logs to a Lambda or ElasticSearch cluster by creating [subscriptions](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Subscriptions.html) on Log Groups.
 * Don't forget to take advantage of the [CloudWatch non-expiring free tier](https://aws.amazon.com/free/#Amazon_CloudWatch).
 
 ### CloudWatch Gotchas and Limitations
 
+* 🔸Metrics in CloudWatch originate [on the hypervisor](https://forums.aws.amazon.com/message.jspa?messageID=403578). The hypervisor doesn't have access to OS information, so certain metrics are not available unless pushed to CloudWatch from inside the instance. Notably RAM utilization.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking phrasing suggestion: "so certain metrics are not available unless pushed to CloudWatch from inside the instance. Notably RAM utilization." -> "so certain metrics (most notably memory utilization) are not available unless pushed to CloudWatch from inside the instance."

Comment (JCBarry) [inline README.md]: This is a relevant document page worth linking to in this Gotcha: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/mon-scripts.html

Comment (JCBarry) [inline README.md]: Ah! I see it mentioned above as well.

**Comment by ThanosBaskous** [comment]
Great! Thanks @bgdnlp! @JCBarry, mind taking a look when you get a chance?

**Comment by ThanosBaskous** [comment]
👍 Thanks all!

**Pull request merged**